### DISCUSSION TITLE
Material Theme UI required ?

### DISCUSSION THREAD

**Discussion by DocAmaroo** [mallowigi/a-file-icon-idea#363] [Q&A] [answered]
Hi,

So I tried your plugin without using Material Theme UI, and it results in a really compacted list view. Meanwhile, using any Material Theme resolve that issue. 

![image](https://user-images.githubusercontent.com/56256989/231121824-8d800358-7168-4c20-9a6c-ecb1f421cb5e.png)

Before to open any bug issue, I was wondering if this plugin was required and this is a normal behavior. Or your plugin can be standalone, and this is should be solved.

Thank you all for your great work.

[Edit] I'm using Webstorm v2023.1 and Atom Material Icons v90.0.0

**Comment by mallowigi** [comment]
No, it's not required, but this should not be the behavior. Did you check the "Custom Line Height" option?

**Comment by mallowigi** [comment, marked as answer]
No, it's not required, but this should not be the behavior. Did you check the "Custom Line Height" option?

**Reply by DocAmaroo** [reply]
That's it ! The option was unchecked.

Thanks mate !

For those looking for it, the full path to the option is :  
`Appareance & Behavior` &rarr; `Atom Material Icons` &rarr; `Settings`  &rarr; `Icon Settings` &rarr; `Size and Line Height`.

**Comment by DocAmaroo** [comment]
Should we close the discussion @mallowigi, or would you like to leave it open?
